test/test_io_opt.py

```diff
@@ -1,7 +1,7 @@
 import unittest
 
 import test_io
-from torchvision import set_video_backend
+from torchvision import set_video_backend  # noqa
```

Comment:
In any case we should be consistent. You removed this line in my comment above, although the setup is the same.